src/paperbot/api/routes/sandbox.py

```
@@ -354,3 +362,194 @@ async def get_system_status(http_request: Request):
+    # Remove from environment
+    if "E2B_API_KEY" in os.environ:
+        del os.environ["E2B_API_KEY"]
+
+    return {"status": "deleted", "message": "E2B API key removed"}
+
+
+# --- Docker Management ---
+
+
+@router.post("/sandbox/docker/start")
+async def start_docker(http_request: Request):
```

Ruff (0.15.1)

[warning] Unused function argument: `http_request`

(ARG001)